extension/src/validator/validator-builder.js

```diff
@@ -10,6 +10,7 @@ function withPayment (paymentObject) {
     validateAdyen () {
       const isAdyen = paymentObject.paymentMethodInfo.paymentInterface === CTP_ADYEN_INTEGRATION
       if (!isAdyen)
+        // todo: should this be an error or just skip if it's not adyen?
```

Comment:
I assume, a question that we answered in the first design of that integration, but yes a valid question for me too.

Reply:
Yes, when I implemented this now, I just noticed this weird thing, so I wanted to raise awareness in the review PR process and to not forget later :) I will test this today if it's working as expected.

Reply:
I checked and this is correct. When we have this error, we ignore the payment. It doesn't fail: https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/src/paymentHandler/payment-handler.js#L21

Reply:
I see a potential refactoring point for that I would not remove that todo maybe just move that functionality to the handler, I assume right now the purpose is not clear because as you said it's not an error, WDYT?